test/spec_helper.rb

```diff
@@ -0,0 +1,84 @@
+SRC_PATH = File.expand_path("../../src", __FILE__)
+DATA_PATH = File.join(File.expand_path(File.dirname(__FILE__)), "data")
+ENV["Y2DIR"] = SRC_PATH
+
+require "yast"
+require "yast/rspec"
+require "update-alternatives/model/alternative"
```

Comment:
why this require is needed here? I expect it is needed only for one test that test alternative, right?